### PR TITLE
Adding new projects at CESNET for pangeo.eu.

### PR DESCRIPTION
**Pull request by rosinec**
Adding new projects at CESNET for pangeo.eu.

### CODE REVIEW

**Comment by rosinec** [comment]
Hi @enolfc, is there a possibility of having more ostack projects for the same VO?

**Comment by enolfc** [comment]
> Hi @enolfc, is there a possibility of having more ostack projects for the same VO?

Yes, we should fix the validation.

File: sites/CESNET-MCC.yaml [hunk 1]
```diff
@@ -117,4 +117,9 @@ vos:
 - name: in-sar-cubes.c-scale.eu
   auth:
     project_id: b28570e8eca54db3b837b85a4f27932f
-  
+- name: vo.pangeo.eu-swift
```
Comment (enolfc) [inline sites/CESNET-MCC.yaml:120]:
```suggestion
- name: /vo.pangeo.eu/swift
```

File: sites/CESNET-MCC.yaml [hunk 2]
```diff
@@ -117,4 +117,9 @@ vos:
...
+  auth:
+    project_id: 57102d3e06b7476088fe4924370ae170
+- name: vo.pangeo.eu-escience
```
Comment (enolfc) [inline sites/CESNET-MCC.yaml:123]:
```suggestion
- name: /vo.pangeo.eu/escience
```

**Review by enolfc — COMMENTED** [review]
This should make the first validation work.

**Review by enolfc — APPROVED** [review]
:+1:

**Pull request merged**